web-ng/src/app/components/map/map.component.ts

```diff
@@ -60,6 +57,8 @@ export class MapComponent implements OnInit, AfterViewInit, OnDestroy {
     streetViewControl: false,
     mapTypeId: google.maps.MapTypeId.HYBRID,
   };
+  private selectedMarker: google.maps.Marker | null = null;
```

Comment:
If we have the option, let's use `selectedMarker?` instead. Then `| null = null` isn't required.

Reply:
It does look cleaner, changed.